src/PrestaShopBundle/Form/FormBuilder.php

```diff
@@ -0,0 +1,91 @@
+<?php
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+declare(strict_types=1);
+
+namespace PrestaShopBundle\Form;
+
+use RuntimeException;
+
+final class FormBuilder extends \Symfony\Component\Form\FormBuilder implements FormBuilderInterface
+{
+    public function addBefore(string $targetFieldName, $child, $type = null, array $options = []): FormBuilderInterface
+    {
+        $this->assertFieldExists($targetFieldName);
```

Comment:
We have a component https://github.com/PrestaShop/PrestaShop/blob/1.7.8.x/src/PrestaShopBundle/Form/FormBuilderModifier.php which does the same but "externally" I believe I copied it from one of your PR in the example module

It's been merged in 178 to ease the module developers extend our form I think it should be used internally in this class by injecting it

Reply:
```suggestion
        $this->modifier->addAfter($this, $targetFieldName, $newChild, $type, $options);
        
        return $this;
```